app/build.gradle

```diff
@@ -378,7 +367,6 @@ dependencies {
     testImplementation Deps.junit
     testImplementation Deps.robolectric
     implementation Deps.fragment_testing
-    testImplementation Deps.megazord_forUnitTests
```

Comment:
IMHO you shouldn't need to list this explicitly as a test dependency in the same way you don't list it explicitly as a build dependency.